### ISSUE TITLE
Duplicate mediators when instantiating gameobject with multiple views

### ISSUE THREAD

**Issue by Nezabyte** [strangeioc/strangeioc#181]
When instantiating a gameobject with 2 views on it, 2 mediators will be added for the first view listed on the game object (the 2nd view will only have 1 associated mediator as expected)

**Comment by fogAndWhisky** [comment]
Thanks for the report.

Does this affect v1.0? Are we talking about 2 Views of the same type, or different types? If the same, can you explain the use case for having two of the same View on one GameObject?

**Comment by Nezabyte** [comment]
Not sure for 1.0; only tested with 0.7 since we planned on sticking with that for now. Will probably switch once 1.0 is merged into master.

Different views.

I went ahead and reproduced the issue with the linked project. Wasn't even necessary to instantiate from a prefab. Seems to have something to do with adding a game object to the scene.

**Steps to reproduce:**
1. Create new Unity3D project
2. Download and import StrangeIOC from the asset store
3. API Update Required dialog - Click 'I made a backup. Go ahead!' 
4. Open scene 'TestView' in folder 'myfirstproject'
5. Duplicate ExampleView and ExampleMediator to create ExampleView2 and ExampleMediator2.
6. Bind ExampleView2 to ExampleMediator2 in context
7. Run scene and click ExampleView gameobject

**Project**
https://dl.dropboxusercontent.com/u/2978478/StrangeIOC_070_TestDuplicateMediators.zip

**Screenshot**
![snag-0120](https://cloud.githubusercontent.com/assets/14398223/10133575/2bae0186-65ac-11e5-8fe3-3cfe269b708b.png)

**Comment by fogAndWhisky** [comment]
@WillzZz Have we already fixed this for v1? It's been lingering around the edges for awhile and it'd be nice to kill it with fire at last.
